difftest/dpi_t1emu/src/dpi.rs

```diff
@@ -236,16 +235,14 @@ unsafe extern "C" fn t1_cosim_final() {
   })
 }
 
-/// evaluate at every 1024 cycles, return reason = 0 to continue simulation,
-/// other value is used as error code.
+/// evaluate at every cycle
+/// return value:
+///   0   : continue
+///   255 : quit successfully
+///   otherwise : error
```

Comment:
Will it slowdown the simulation?

Reply:
My intuition is that a small function call every cycle won't slowdown. I could do some measurements later.